### ISSUE TITLE
Small objects in large environment

### ISSUE THREAD

**Issue by jax79sg** [junyanz/CycleGAN#88]
Hi, 

Thanks for making this available.

I am trying out styles transfer (Normal and Night mode) for a off shore environment. The objects in focus are ships and thus quite small on each frame. I tried having equal --load_size and --fine_size so the objects are not cropped out randomly but the trained model still couldn't produce the fake ships. The fake sea and sky color looks good though. How might we improve on this?

Thanks.
Regards.

**Comment by junyanz** [comment]
I think you may want to use a smaller fine_size.

**Comment by jax79sg** [comment]
Hi thanks for the reply.

Please correct me if I am wrong, is the fine size parameter meant to randomly crop the image into a number of smaller images, with fine size being the size of these cropped images? If that's so, should I set the fine size to size of the typical smallest objects in the image?

Thanks.

**Comment by junyanz** [comment]
Yes. fineSize is the size of cropped images. Also, see more options for [resize_or_crop](https://github.com/junyanz/CycleGAN/blob/83077c9d40d729ca0dbb7ff029b90a507d32accd/options.lua#L51).

**Comment by zengxiaoliang** [comment]
> Hi,
> 
> Thanks for making this available.
> 
> I am trying out styles transfer (Normal and Night mode) for a off shore environment. The objects in focus are ships and thus quite small on each frame. I tried having equal --load_size and --fine_size so the objects are not cropped out randomly but the trained model still couldn't produce the fake ships. The fake sea and sky color looks good though. How might we improve on this?
> 
> Thanks.
> Regards.



Hi, did U  transfer styles success （Normal to Night）？
